header-link.js

```diff
@@ -0,0 +1,6 @@
+let elements = document.getElementsByClassName("header-copy");
+for (let i = 0; i < elements.length; i++) {
+    let element = elements[i];
+    let parentId = element.parentElement.id;
+    addCopyButton(element, location.protocol + '//' + location.host + location.pathname + "#" + parentId)

```

Comment:
only works if the it is hosted at the root level (i.e. not at example.com/tbfightersdotorg/)